app/models/case_log.rb

```diff
@@ -465,6 +465,14 @@ def location_code
     location&.id
   end
 
+  def la_label
```

Comment:
Yep they're still necessary, otherwise this error surfaces on csv download: 
NoMethodError in CaseLogsController#index
undefined method `la_label' for #<CaseLog...

Reply:
Was thinking we could just call these bits directly there but not super important